src/iopae.cpp

```diff
@@ -539,9 +539,10 @@ void PAEOutput::WriteDur(DurationInterface *interface)
 {
     assert(interface);
 
-    if ((interface->GetDur() != m_currentDur) || (interface->GetDots() != m_currentDots)) {
+    int m_dots = (interface->HasDots()) ? interface->GetDots() : 0;
```

Comment:
local variables are not prefixed with `m_`

Reply:
(And in cases like these we usually make them const.)